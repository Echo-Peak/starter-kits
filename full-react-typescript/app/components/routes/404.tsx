import * as React from 'react';
const {Component} = React;

interface Iprops {}
interface Istate {}

export default class NotFound extends Component<Iprops, Istate>{
    render(){
        return (
            <div>
              <h1>Not found</h1>
            </div>    
        )
    }
}