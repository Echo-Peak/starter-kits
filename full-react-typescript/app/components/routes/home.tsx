import * as React from 'react';
const {Component} = React;

interface Iprops {}
interface Istate {}

export default class Home extends Component<Iprops , Istate>{
    render(){
        return <h1>Homepage</h1>
    }
}