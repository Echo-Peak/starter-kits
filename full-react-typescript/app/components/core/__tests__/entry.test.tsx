jest.dontMock('react');
jest.dontMock('react-dom');
jest.dontMock('react-addons-test-utils');

import * as React from 'react';
import {Router, browserHistory} from 'react-router';
import {render , mount} from 'enzyme';
import routes from '../router';
import Home from '../../routes/home';
import NotFound from '../../routes/404';


describe('Application bootstrap', () => {

  it('bootstaps the application, sets up the router with "/" being the index route', (done) => {
    
    const component = mount(
          <div>
        <Router history={browserHistory}>
            {routes}
        </Router>
    </div> 
    );

    done();
  });
});
describe('converts', () => {

  it('converts', (done) => {
    
    const data = JSON.stringify({one:283});
    done()
  });
});