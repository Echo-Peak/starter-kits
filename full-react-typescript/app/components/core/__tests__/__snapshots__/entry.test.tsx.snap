// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`converts converts 1`] = `"{\\"one\\":283}"`;
